src/common/massert.h

```diff
@@ -25,8 +25,7 @@
 #include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
-
-#include "common/sfserr.h"
+#include <sys/syslog.h>
```

Comment:
Unneeded change.

Reply:
This is definitely needed, because the macros below require it. I'm guessing it's showing up as unused because of the fact they are per-processor macros.

Reply:
The other alternative is to include them where the macros are used, but I think it's easiest this way to already include the requirements with this header file.